cmd/cve/generate.go

```diff
@@ -44,11 +45,22 @@ func main() {
 		panic(err)
 	}
 
+	// Download the last released NVD feed
+	logger.Log("msg", "Downloading latest release")
+	if err := downloadLatestRelease(*dbDir, *debug, logger); err != nil {
```

Comment:
Great idea!

I'm guessing that if there's newer updates in the NVD site those are pulled in? (because we release the `last_mod_start_date.txt`).

Reply:
correct, once we pull down the last release feed we update as we today on fleet servers